app/views/book-appointment.nunjucks

```diff
@@ -1,5 +1,95 @@
 {% extends 'layout.nunjucks' %}
 
+{% block headerClass %}local-header{% endblock %}
+
+{% block header %}
+<h1 class="local-header__title">Book a GP appointment at {{gp.name}}</h1>
+{% endblock %}
+
+{% block pageIntro %}
+<div class="local-header__intro">
+  <div class="reading-width">
+    <ul class="link-list">
+      <li><a href="#contact-reception">Contact reception</a></li>
+      <li><a href="#book-appointment">Book an appointment online</a></li>
+    </ul>
+  </div>
+</div>
+{% endblock %}
+
 {% block content %}
-Book an appointment at {{gp.name}}
+<div class="reading-width">
+  <h2 id="contact-reception">Contact reception</h2>
+  <h3>Book an appointment</h3>
+  <p>Contact reception: {{gp.phone}}</p>
```

Comment:
The phone number is showing as empty on the page. In the latest schema the phone number has moved to 'contact.telephone', so needs to change if profiles 0.2.0 is being used.